File: app_java/src/main/java/vlab/android/architecture/base/BaseActivity.java
package vlab.android.architecture.base;

import android.os.Bundle;

import vlab.android.common.ui.DaggerCommonActivity;
import vlab.android.common.util.LogUtils;

/**
 * Created by Vinh Tran on 2/15/18.
 */

public class BaseActivity extends DaggerCommonActivity {

    @Override
    public void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        LogUtils.d(getClass().getSimpleName(), ">>> onCreate");
    }
}

File: app_kotlin/src/main/java/vlab/android/architecture/base/BaseActivity.kt
package vlab.android.architecture.base

import android.os.Bundle
import vlab.android.common.ui.CommonActivity
import vlab.android.common.util.LogUtils

/**
 * Created by Vinh Tran on 2/15/18.
 */

open class BaseActivity : CommonActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        LogUtils.d(javaClass.simpleName, ">>> onCreate")
    }
}